test_login.py

```diff
@@ -0,0 +1,25 @@
+import requests
+
+base_url = 'https://python-qa-training.herokuapp.com/api/auth/login'
+email = 'sievdokymov@lohika.com'
+password = '12345678'
+
```

Comment:
According to PEP-8, it requires 2 empty lines between the last attribute and function. The same between all functions (tests) in the module.

More about PEP-8 - https://www.python.org/dev/peps/pep-0008/

Reply:
@bochkala fixed in latest version submitted as "homework1-5" pull request - appropriate code style changes were made